en/newbee/mod.md

```diff
@@ -2,47 +2,47 @@
 sidebarDepth: 2
 ---
 
-# 配置 Mod
+# Mod Configurations
 
-LittleSkin 仅提供材质的上传、存储、检索和分享的功能。想要在 Minecraft 中显示你在 LittleSkin 设置的材质的话，你需要在 Minecraft 客户端中安装皮肤 Mod 并修改相应的配置文件。
+LittleSkin will only provide the functionality of uploading, storage, searching and sharing. If you want to use the textures applied in LittleSkin in Minecraft, you have to install skin loader mods in Minecraft Client and change related configuration files.
 
 :::tip
-如何安装 Mod 请自行搜索，本文不会说明如何安装皮肤 Mod，仅说明如何配置皮肤 Mod，使其从 LittleSkin 加载材质。
+Please make sure that you know how to install mods. This documentation will not explain how to install mods, but just how to configure them and allow them to load textures from LittleSkin.
 
-在一些情况下，安装皮肤 Mod 后，可能需要启动一次游戏并进入存档，Mod 才会自动生成配置文件；如果你在启动器中启用了版本隔离，配置文件的路径可能有所不同。
+In some cases, after you install your mod, you may have to launch the game and load a save to allow the mod to generate the configuration file. If you enabled the version isolation in your launcher, the file path of the configuration file may change.
 :::
 
 ::: danger
-只需要使用一种皮肤 Mod 即可。请不要同时安装多个皮肤 Mod，否则，轻则无法正常加载材质，重则导致游戏崩溃。
+You should only use one skin loader mod. Do not install multiple skin mods, otherwise, the skin might not load normally, it could even crash your game.
 :::
 
 ::: warning
-除 SkinPort 外，皮肤 Mod 和 Yggdrasil 外置登录二选一即可。请不要同时使用这两者，否则可能无法正常加载材质。
+Except for SkinPort, you should only use a skin mod or Yggdrasil. Do not enable both at the same time, otherwise, the skin might not load normally.
```

Comment:
you should only use either one skin mod or authlib-injector (Yggdrasil API). **DO NOT** use multiple methods or mods at the same time,